src/scss/features.scss

```diff
@@ -0,0 +1,245 @@
+        color: rgb(34, 34, 34);
+        line-height: 0.75;
+        text-align: center;
+    }
+
+    &_icon-description {
+        font-size: 18px;
+        font-family: "HelveticaNeue";
+        color: rgb(132, 132, 132);
+        line-height: 1.625;
+        text-align: center;
+        margin-top: 20px;
+    }
+
+    &_col {
+        margin: 0;
+        padding: 0;
+    }
+
+    &_1 {
+        &::after {
+            content : "";
+            position: absolute;
+            left    : 10%;
+            bottom  : 0;
+            height  : 1px;
+            width   : 80%;
+            border-bottom: 1px solid $lightgray;
+        }
+
+        &::before {
+            @include for-sm-up {
+                content : "";
+                position: absolute;
+                top    : 10%;
+                right  : 0;
+                height  : 1px;
+                height   : 80%;
+                border-right: 1px solid $lightgray;
+            }
+        }
+    }
+
+    &_2 {
+        &::after {
+            content : "";
+            position: absolute;
+            left    : 10%;
+            bottom  : 0;
+            height  : 1px;
+            width   : 80%;
+            border-bottom: 1px solid $lightgray;
+        }
+
+        &::before {
+            @include for-md-up {
+                content : "";
+                position: absolute;
+                top    : 10%;
+                right  : 0;
+                height  : 1px;
+                height   : 80%;
+                border-right: 1px solid $lightgray;
+            }
+        }
+    }
+
+    &_3 {
+        &::after {
+            content : "";
+            position: absolute;
+            left    : 10%;
+            bottom  : 0;
+            height  : 1px;
+            width   : 80%;
+            border-bottom: 1px solid $lightgray;
+        }
+
+        &::before {
+            @include for-sm-up {
+                content : "";
+                position: absolute;
+                top    : 10%;
+                right  : 0;
+                height  : 1px;
+                height   : 80%;
+                border-right: 1px solid $lightgray;
+            }
+
+            @include for-md-up {
+                border-right: 0;
+            }
+        }
+    }
+
+    &_4 {
+        &::after {
+            content : "";
+            position: absolute;
+            left    : 10%;
+            bottom  : 0;
+            height  : 1px;
+            width   : 80%;
+            border-bottom: 1px solid $lightgray;
+            
+            @include for-md-up {
+                border-bottom: 0;
+            }
+        }
+
+        &::before {
+            @include for-md-up {
+                content : "";
+                position: absolute;
+                top    : 10%;
+                right  : 0;
+                height  : 1px;
+                height   : 80%;
+                border-right: 1px solid $lightgray;
+            }
+        }
+    }
+
+    &_5 {
```

Comment:
I noticed that this section (`_1`, `_2` and so on) has a lot of duplicated code (`position`, `margin` and so on). What could you have done differently to reduce this duplicated code?

Reply:
That is indeed true. One alternative option would be creating SASS mixins with an argument (to be able to insert or remove the border):

```css

@mixin pseudo-border-bottom($insert: true) {

    @if $insert {
        content : "";
        position: absolute;
        left    : 10%;
        bottom  : 0;
        height  : 1px;
        width   : 80%;
        border-bottom: 1px solid $lightgray;
    } @else {
        border-bottom: 0;
    }
}

@mixin pseudo-border-right($insert: true) {

    @if $insert {
        content : "";
        position: absolute;
        top    : 10%;
        right  : 0;
        height  : 1px;
        height   : 80%;
        border-right: 1px solid $lightgray;
    } @else {
        border-right: 0;
    }
}
```

By using that, the code would look like this:

```css
&_1 {
    &::after {
        @include pseudo-border-bottom();
    }

    &::before {
        @include for-sm-up {
            @include pseudo-border-right();
        }
    }
}

&_2 {
    &::after {
        @include pseudo-border-bottom();
    }

    &::before {
        @include for-md-up {
            @include pseudo-border-right();
        }
    }
}

&_3 {
    &::after {
        @include pseudo-border-bottom();
    }

    &::before {
        @include for-sm-up {
            @include pseudo-border-right();
        }

        @include for-md-up {
            @include pseudo-border-right(false);
        }
    }
}

&_4 {
    &::after {
        @include pseudo-border-bottom();
        
        @include for-md-up {
            @include pseudo-border-bottom(false);
        }
    }

    &::before {
        @include for-md-up {
            @include pseudo-border-right();
        }
    }
}

&_5 {
    &::after {
        @include for-xs-only {
            @include pseudo-border-bottom();
        }
    }

    &::before {
        @include for-sm-up {
            @include pseudo-border-right();
        }
    }
}
```